src/core/image/BackgroundImage.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+
+import testImage from '../../common/utils/testImage'
+import styles from './BackgroundImage.module.scss';
+import { Tag } from '../../common/components/tag/Tag';
+import useBackgroundFallbackImageUrl from '../hooks/useBackgroundFallbackImageUrl';
+
+type Props = {
+  url: string,
+  customFallbackUrl?: string,
+  labelTag?: string,
+} & React.HTMLAttributes<HTMLDivElement>
+
+export default function BackgroundImage({
+                                url,
+                                customFallbackUrl,
+                                labelTag,
+                                children,
+                                ...divProps
+                             }: Props) {
+
+  const [showBackupImage, setShowBackupImage] = React.useState(false);
+  const defaultFallbackUrl = useBackgroundFallbackImageUrl(url);
+  const fallbackUrl =  customFallbackUrl ?? defaultFallbackUrl;
+  React.useEffect(() => {
+    const testThatImageExist = async () => {
+      try {
+        await testImage(url);
+      } catch {
+        setShowBackupImage(true);
+      }
+    };
+    testThatImageExist();
+  }, [url]);
+
+  return <div
```

Comment:
is there a change that we have a delay when testing the image, and it will be some ui unexpected state (like jumping elements because of the height value change) or its handled somehow in the styles?

Reply:
I haven't tested it throughfully yet, perhaps we should imitate the slow connection speed from browser settings or so.